frontend/src/app/spot/styles/sass/components/list.sass

```diff
@@ -37,9 +37,18 @@
       &:not(:last-child)
         margin-left: $spot-spacing-0_5
 
+  &_compact &--item:not(:first-child)
+    margin-top: $spot-spacing-0_25
```

Comment:
Why do the compact items need a margin-top if the normal list doesn't? Could we handle that with the paddings like in the normal list?

Reply:
It was done because otherwise the list was **too** compact :D Inreasing the paddings doesn't work because we don't have an intermediate value for these paddings as a spacing. (e.g. no `spot-spacing-0_375`) If we'd add a different margin on the top than the bottom the hover would feel misaligned.

Reply:
Is there anything left to be done on those spacings, @HDinger and @b12f ? Or is this PR rtm?

Reply:
Ready to merge for me